ui/app/src/components/UsersGrid/UsersGrid.tsx

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
We should split this one up in the same way as sites: SitesManagement & SitesGrid